stable_plugins/classical_ml/scikit_ml/classical_k_means/tasks.py

```diff
@@ -83,13 +84,15 @@ def calculation_task(self, db_id: int) -> str:
             title=f"Classical {num_clusters}-Means Clusters",
         )
 
+    info_str = f"_clusters_{num_clusters}"
```

Comment:
maybe also include the algorithm: `_k-means_clusters_{num_clusters}` (or in this case maybe even `_{num_clusters}-means`)
Also relevant for the other filenames.